### PR TITLE
C33632 Verify code by sms page is displayed

### PR DESCRIPTION
**Pull request by angy113**

### CODE REVIEW

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/hooks/PinLayoutHook.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.jalasoft.callingapp.hooks;
+
+import com.jalasoft.callingapp.ui.layouts.pages.Login.PinVerificationLayout;
+import cucumber.api.java.After;
+
+public class PinLayoutHook {
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/java/com/jalasoft/callingapp/hooks/PinLayoutHook.java]: add doc for the class

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java [hunk 2]
```diff
@@ -41,9 +49,50 @@ public void iLogInToTheApplication(final Map<String, String> loginCredentialsMap
...
+        }
+    }
+
+    /**
+     * Verifies if " Verify Mu Number " is disabled.
+     */
+    @Then("^The Verify My Number button should be (enabled|disabled)$")
+    public void theVerifyMyNumberButtonShouldBeEnabledDisabled(final String status) {
+        if (status.equals("disabled")) {
+            assertFalse(loginLayout.isVerifyMyNumberButtonEnabled());
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: change the asserts to:
assertFalse(validation, "message");

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java [hunk 3]
```diff
@@ -41,9 +49,50 @@ public void iLogInToTheApplication(final Map<String, String> loginCredentialsMap
...
+        } else {
+            assertTrue(loginLayout.isVerifyMyNumberButtonEnabled());
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: same here

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java [hunk 4]
```diff
@@ -41,9 +49,50 @@ public void iLogInToTheApplication(final Map<String, String> loginCredentialsMap
...
+       // pinVerificationLayout.waitForVerificationCodeFilled();
+    }
+
+    /**
+     * Verifies if an specific message is displayed.
+     * @param textMessage the text message to verify if it is displayed.
+     */
+    @Then("^The \"([^\"]*)\" message should be displayed$")
+    public void theMessageShouldBeDisplayed(final String textMessage) {
+        assertTrue(pinVerificationLayout.isTextMessageDisplayed(textMessage));
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: Same here

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/hooks/PinLayoutHook.java (hunk 1 above)
Comment (angy113) [reply IDT.CallingApp/src/test/java/com/jalasoft/callingapp/hooks/PinLayoutHook.java]: Done

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java (hunk 3 above)
Comment (angy113) [reply IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: Done

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java [hunk 5]
```diff
@@ -59,7 +69,7 @@ public void iLogInToTheApplication(final Map<String, String> loginCredentialsMap
     @Given("^I am not login to the application$")
     public void iAmNotLoginToTheApplication() {
         if (!loginLayout.isDisplayed()) {
-            BaseAppLayout baseAppLayout = new BaseAppLayout();
+            BaseAppLayout baseAppLayout= new BaseAppLayout();
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: add space before =

**Pull request merged**